__test__/integration/debt_token_api/scenarios/successful_transfer_from.ts

```diff
@@ -0,0 +1,117 @@
+// External
+import { BigNumber } from "bignumber.js";
+
+// Utils
+import * as Units from "utils/units";
+import { ERC20TokenSymbol } from "utils/constants";
+import { ACCOUNTS } from "../../../accounts";
+
+// Types
+import { DebtTokenScenario } from "./";
+import { SimpleInterestLoanOrder } from "src/adapters/simple_interest_loan_adapter";
+
+const CREDITOR = ACCOUNTS[0].address;
+const DEBTOR = ACCOUNTS[1].address;
+const TOKENS_APPROVED_OPERATOR = ACCOUNTS[2].address;
+const OWNERS_APPROVED_OPERATOR = ACCOUNTS[3].address;
+
+const ORDER_1: SimpleInterestLoanOrder = {
+    principalAmount: Units.ether(10),
+    principalTokenSymbol: ERC20TokenSymbol.ZRX,
+    interestRate: new BigNumber(4.135),
+    amortizationUnit: "months",
+    termLength: new BigNumber(3),
+    debtor: DEBTOR,
+    creditor: CREDITOR,
+};
+
+const ORDER_2: SimpleInterestLoanOrder = {
+    principalAmount: new BigNumber(11 * 10 ** 18),
+    principalTokenSymbol: ERC20TokenSymbol.MKR,
+    interestRate: new BigNumber(8.937),
+    amortizationUnit: "years",
+    termLength: new BigNumber(2),
+    debtor: DEBTOR,
+    creditor: CREDITOR,
+};
+
+const defaults: DebtTokenScenario.TransferFromScenario = {
+    // Scenario Setup Parameters
+    description: `Default Description`,
```

Comment:
why not just make this a vanilla object and exclude description. there's no scenario where you should rely on a default description...